### PR TITLE
Stable version promise async wait chrome headless

### PR DESCRIPTION
**Pull request by jcarrillog-psl**
Version stable working on travis.
image

### CODE REVIEW

File: protractor.config.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 exports.config = {
 	framework: 'jasmine' ,
-	seleniumAddress: 'http://localhost:4444/wd/hub' ,
+	//seleniumAddress: 'http://localhost:4444/wd/hub' ,
```
Comment (aperdomob) [inline protractor.config.js]: eliminar el comentario

**Review by aperdomob — CHANGES_REQUESTED** [review]

**Review by aperdomob — APPROVED** [review]

**Review by aperdomob — CHANGES_REQUESTED** [review]
sorry delete the comment line :-p

File: protractor.config.js (hunk 1 above)
Comment (jcarrillog-psl) [reply protractor.config.js]: Done!

**Comment by jcarrillog-psl** [comment]
> sorry delete the comment line :-p

Done!

**Pull request merged**